.github/workflows/pages.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: |
           mkdir docs/
           pip install -r rules_inventory/scripts/requirements.txt
-          python rules_inventory/scripts/rules_overview_generator.py --rules_file=rules/falco_rules.yaml > docs/overview.md
+          python rules_inventory/scripts/rules_overview_generator.py --rules_file=rules/falco_rules.yaml > docs/index.md
```

Comment:
As per mkdocs documentation, default page must be named either `index.md` or `README.md`